google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadCallConversion.java

```diff
@@ -0,0 +1,184 @@
+      return;
+    }
+
+    try {
+      new UploadCallConversion()
+          .runExample(
+              googleAdsClient,
+              params.customerId,
+              params.conversionActionId,
+              params.callerId,
+              params.callStartDateTime,
+              params.conversionDateTime,
+              params.conversionValue);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the client.
+   * @param customerId the customer ID.
+   * @param conversionActionId the conversion action ID.
+   * @param callerId the caller ID.
+   * @param callStartDateTime the call start date time.
+   * @param conversionValue the value of the conversion in USD.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient,
+      long customerId,
+      String conversionActionId,
+      String callerId,
+      String callStartDateTime,
+      String conversionDateTime,
+      double conversionValue) {
+    // Create a call conversion by specifying currency as USD.
+    CallConversion conversion =
+        CallConversion.newBuilder()
+            .setConversionAction(StringValue.of(conversionActionId))
+            .setCallerId(StringValue.of(callerId))
+            .setCallStartDateTime(StringValue.of(callStartDateTime))
+            .setConversionDateTime(StringValue.of(conversionDateTime))
+            .setConversionValue(DoubleValue.of(conversionValue))
+            .setCurrencyCode(StringValue.of("USD"))
+            .build();
+
+    // Uploads the call conversion to the API.
+    try (ConversionUploadServiceClient conversionUploadServiceClient =
+        googleAdsClient.getLatestVersion().createConversionUploadServiceClient()) {
+      // Partial failure MUST be enabled for this request.
+      UploadCallConversionsResponse response =
+          conversionUploadServiceClient.uploadCallConversions(
+              UploadCallConversionsRequest.newBuilder()
+                  .setCustomerId(String.valueOf(customerId))
+                  .addAllConversions(ImmutableList.of(conversion))
+                  .setPartialFailure(true)
+                  .build());
+
+      // Prints any partial failure errors returned.
+      if (response.hasPartialFailureError()) {
+        throw new RuntimeException(
```

Comment:
Our other examples print errors instead of throw them. Did you do this as part of the move to help us better test code examples and detect failures?